examples/cordova-sample/MyApp/www/js/index.js

```diff
@@ -102,8 +48,64 @@ const app = {
     console.log("Received Event: " + id);
     console.log("---------");
     Purchases.setDebugLogsEnabled(true);
-    Purchases.configure("api_key");
-  }
+    Purchases.configure("api_key", function() { 
+      console.log("Configured");
```

Comment:
Maybe a bit more detail?
```suggestion
      console.log("RC SDK Configured");
```

Reply:
yeah I phoned it in didn't I. I'll clean it up, thanks!